postgres/ppg_builder.sh

```diff
@@ -107,6 +107,9 @@ get_sources(){
 	sed -i 's:enable_tap_tests=no:enable_tap_tests=yes:' configure
 	if [ -d "contrib/pg_tde" ]; then
             WITH_TDE=1
+            cd contrib/pg_tde
+            git pull origin main
```

Comment:
Maybe better to use branch as a variable?

Reply:
@EvgeniyPatlan Okay. Introduced variable TDE_BRANCH for branch name. BTW these are just temporary changes and we will remove them when TDE will be ready for release. This is the reason I have not added new parameter to builder script.

Reply:
Reverted all changes because they were not required